pypesto/ensemble/utils.py

```diff
@@ -57,6 +59,50 @@ def read_from_csv(path: str,
                         upper_bound=upper_bound)
 
 
+def read_ensemble_from_hdf5(filename: str,
+                            input_type: str = OPTIMIZE,
+                            remove_burn_in: bool = True,
+                            chain_slice: slice = None,
+                            cutoff: float = np.inf,
+                            max_size: int = np.inf
+                            ):
+    """
+    Create an ensemble from an HDF5 storage file.
+
+    Parameters:
+    ----------
+    filename:
+        Name or path of the HDF5 file.
+    input_type:
+        Which type of ensemble to create. From History, from
+        Optimization or from Sample.
+
+    Returns:
+    -------
+    ensemble:
+        Ensemble object of parameter vectors
+    """
+    # TODO: add option HISTORY. Need to fix
+    #  reading history from hdf5.
+    if input_type == OPTIMIZE:
+        result = read_result(filename=filename,
+                             optimize=True)
+        return Ensemble.from_optimization_endpoints(result=result,
+                                                    cutoff=cutoff,
+                                                    max_size=max_size)
+    elif input_type == SAMPLE:
+        result = read_result(filename=filename,
+                             sample=True)
+        return Ensemble.from_sample(result=result,
+                                    remove_burn_in=remove_burn_in,
+                                    chain_slice=chain_slice)
+    else:
+        raise ValueError('The type you provided was neither '
+                         '"sample" nor "optimization". Those are '
```

Comment:
use string constnts here

Reply:
dmn, now on my keybord lso the _ key is broken. rrg tht's wful.

Reply:
I can still read it 👍 so define this string in the `constants.py`?

Reply:
It is e.g. `SAMPLE == 'sample'`, right? Then I'd just use `f'{SAMPLE} nor {OPTIMIZATION}. Those are '` here.

Reply:
```suggestion
                         f'"{SAMPLE}" nor "{OPTIMIZE}". Those are '
```